kubens

```diff
@@ -173,7 +173,9 @@ swap_namespace() {
 }
 
 main() {
-  if hash kubectl 2>/dev/null; then
+  if [[ "${KUBECTL:-}" ]]; then
```

Comment:
I don't know what `[[ "$var" ]]` test does (even though manpage says it checks string isn't null), I recommend you stick to -n or -z here.

Not sure what the next line saying `true` does either.

I recommend if you rewrite this like

```
if [[ -z "${KUBECTL:-}" ]] then;
  if hash ..
  elif ..
  else ..
  fi

fi
```